function f() {
	throw 42;
}

function main() {
	f();
}
except(e) {
	std::println(e);
}